assets/javascripts/kitten/components/icons/arrow-icon/stories.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { Grid, GridCol } from '../components/grid/grid'
```

Comment:
Je pense que le déploiement Vercel marchera comme ça :

```suggestion
import { Grid, GridCol } from '../../../components/grid/grid'
```

Reply:
Ah bien vu ! Merci